projects/arlas-toolkit/src/lib/components/bookmark/bookmark-add-dialog.component.html

```diff
@@ -1,12 +1,14 @@
 
-<div mat-dialog-content class="add_bookmark">
-  <h1 class="modal_title" mat-dialog-title>{{ 'Dataset name' | translate }} </h1>
-  <mat-form-field>
-    <input matInput tabindex="1"  [formControl]="bookmarkName" class="bookmark-search-input" (keypress)="pressEvent($event)">
-    <mat-error *ngIf="bookmarkName.invalid">{{errorMessage}}</mat-error>
+<div mat-dialog-content class="bookmark__add">
+  <h1 class="bookmark__add--title" mat-dialog-title>
+    {{ 'Dataset name' | translate }}
```

Comment:
Normally we renamed all the places were it was 'Dataset' by Bookmar. I think this title was not changed precedently

Reply:
In the i18n file I still have all the keys with dataset and not bookmark. I think we just changed the translation

Reply:
ok !